src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.scss

```
@@ -52,4 +53,11 @@
 input:-webkit-autofill {
   animation-name: autofill;
   animation-fill-mode: both;
+  box-shadow: 0 0 0px 1000px $color-password-toggle-bg inset;
+  -webkit-box-shadow: 0 0 0px 1000px $color-password-toggle-bg inset;
+  -webkit-text-fill-color: $color-password-toggle-text;
+  transition-property: none;
+  -webkit-transition-property: none;
+  border-radius: 0px;
+  -webkit-border-radius: 0px;
```

```suggestion
  box-shadow: 0 0 0px 1000px $color-password-toggle-bg inset;
  -webkit-text-fill-color: $color-password-toggle-text;
  transition-property: none;
  border-radius: 0px;
```
by default `-webkit` is no longer needed.

Is `-webkit-text-fill-color` really needed? Documentation says `This feature is non-standard and should not be used without careful consideration.`, so just wondering if it can be removed or not.

---

Without `-webkit-text-fill-color` the `color` of the text inside input box changes to black during autocomplete.

---

Thanks for the info.